flaml/automl/model.py

```diff
@@ -2028,6 +2028,104 @@ def fit(self, X_train, y_train, budget=None, free_mem_ratio=0, **kwargs):
         return train_time
 
 
+class HoltWinters(ARIMA):
+    """
+    The class for tuning Holt Winters model, aka 'Triple Exponential Smoothing'.
+    """
+
+    @classmethod
+    def search_space(cls, **params):
+        space = {
+            "damped_trend": {
+                "domain": tune.choice([True, False]),
+                "init_value": False,
+                "low_cost_init_value": False,

```

Comment:
"low_cost_init_value" is not needed here. "init_value" is enough.

Reply:
good point, this model is very cheap to fit. Thank you